kubernetes-client/src/test/resources/test-kubeconfig-exec

```diff
@@ -1,4 +1,4 @@
-apiVersion: v1
+apiGroupVersion: v1
```

Comment:
#1347: this does not look right. An actual `~/.kube/config` uses `apiVersion`, not `apiGroupVersion`. So the test is being updated to match the new broken behavior?

Reply:
Right, this is wrong.